templates/supertest/put/put.handlebars

```diff
@@ -1,4 +1,4 @@
-  it('should respond with {{length description}}', function(done) {
+  it('should respond with {{length description}}{{#if inputs}}, {{inputs.body.message}}{{/if}}', function(done) {
```

Comment:
Hmm, maybe I don't get the idea, but why would we alter the descriptions? That is what's defined in the swagger json right?
Also, where does this message come from? Not from the backend, so we define extra messages in the mock data?
But like I said, maybe I'm just missing something. In that case: help me out! :-)